ql/src/test/queries/clientpositive/materialized_view_rewrite_1.q

```diff
@@ -5,6 +5,7 @@ set hive.txn.manager=org.apache.hadoop.hive.ql.lockmgr.DbTxnManager;
 set hive.strict.checks.cartesian.product=false;
 set hive.stats.fetch.column.stats=true;
 set hive.materializedview.rewriting=true;
+set hive.materializedview.rewriting.query.text=false;
```

Comment:
Why is this disabled now? Does the test change?

Reply:
This test was written for calcite based rewrite